### PR TITLE
Clicking 'add metadata' when files are uploading breaks the metadata

### PR DESCRIPTION
**Pull request by StefanBabukov**
# Background
#### Link to issue 
https://biomage.atlassian.net/secure/RapidBoard.jspa?rapidView=1&projectKey=BIOMAGE&modal=detail&selectedIssue=BIOMAGE-1048
#### Link to staging deployment URL 
https://ui-stefan-ui341.scp-staging.biomage.net/
#### Anything else the reviewers should know about the changes here

# Changes
### Code changes

# Definition of DONE
Your changes will be ready for merging after each of the steps below have been completed:

### Testing
- [ ] Unit tests written
- [ ] Tested locally with Inframock (with latest production data downloaded with biomage experiment pull)
- [ ] Deployed to staging

To set up easy local testing with inframock, follow the instructions here: https://github.com/biomage-ltd/inframock
To deploy to the staging environment, follow the instructions here: https://github.com/biomage-ltd/biomage-utils

### Documentation updates
Is all relevant documentation updated to reflect the proposed changes in this PR?

- [ ] Relevant Github READMEs updated
- [ ] Relevant wiki pages created/updated

### Approvers
- [ ] Approved by a member of the core engineering team
- [ ] (UX changes) Approved by vickymorrison (this is her username, tag her if you need approval)

### Just before merging:
- [ ] After the PR is approved, the `unstage` script in here: https://github.com/biomage-ltd/biomage-utils is executed. This script cleans up your deployment to staging

### Optional
- [ ] Photo of a cute animal attached to this PR

### CODE REVIEW

File: src/components/data-management/ProjectDetails.jsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import React, { useState, useEffect, useRef } from 'react';
+import _ from 'lodash';
```
Comment (aerlaut) [inline src/components/data-management/ProjectDetails.jsx]: I don't see where `_` is used. I think you can remove this.

**Review by aerlaut — APPROVED** [review]
Code looks good to me. You can merge in after removing the unused `lodash` import. Tested and it works. Can you explain how this fix works?

**Comment by vickymorrison** [comment]
I can't test if it's fixed in the staging link because this error happens when trying to upload files to a new project. Is this expected?
Screenshot 2021-06-18 at 08 41 10

File: src/components/data-management/ProjectDetails.jsx (hunk 1 above)
Comment (StefanBabukov) [reply src/components/data-management/ProjectDetails.jsx]: When clicking `add metadata` a new column is added to the table. But when uploading a file the table is updated all the time and you can see that the old changes were setting the table columns to the initial ones every time which removes the metadata popover.

**Comment by vickymorrison** [comment]
You can ignore my comment above ^^ - it's an issue with using Firefox (I'll make a ticket for it). Your fix works in Chrome.

**Review by vickymorrison — APPROVED** [review]

**Pull request merged**